libs/react-components/community/src/components/toggle-open/toggle-open.tsx

```diff
@@ -23,20 +23,35 @@ export interface ToggleOpenProps extends Omit<ButtonProps, 'children' | 'iconRig
    * Overidden from ButtonProps, because we dont support string as icon name here
    */
   iconRight?: Partial<IconProps>;
+  /**
+   * If the element is a button or a div
+   * @default button
+   */
+  as?: 'button' | 'div';
```

Comment:
PolymorphicComponentPropWithRef types should probaly used here, thought nothing is broken with it.
Poly props are not very straightforward to use, but Sidenav for example has example of it.
@airikej is it ok?

Reply:
Yes I think its fine for now, ready to merge!